docs/kratos/social-signin/65_netid.mdx

```diff
@@ -4,12 +4,71 @@ title: Add NetID as a social sign-in provider in Ory
 sidebar_label: NetID
 ---
 
+```mdx-code-block
+import JsonnetWarning from "../../_common/jsonnetwarning.mdx"
+```
+
 # NetID
 
+````mdx-code-block
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<Tabs>
+<TabItem value="console" label="Ory Console" default>
+
+Follow these steps to add NetID as a social sign-in provider to your project using the Ory Console:
+
+1. Sign in to Ory Console and select [**Social Sign-in**](https://console.ory.sh/projects/current/social-signin).
+2. Click the switch next to the NetID logo to start the configuration.
+3. Copy the **Redirect URI** and save it for later use.
+4. Create a [NetID Service](https://developerzone.netid.dev/latest/devportal/tutorial/services/) and
+   [NetID Client](https://developerzone.netid.dev/latest/devportal/tutorial/clients/).
+5. While creating the NetID Client, enter the saved **Redirect URI** from Ory to the **Callback URL** field of the NetID Client.
```

Comment:
```suggestion
5. Enter the Redirect URI you copied from Ory ine the **Callback URL** field of the NetID Client.
```

Please use structure like this. Simpler, shorter.

Reply:
The reason for the "While" clause is that there is a bit of a contradiction in sequencing here: if you have already finished creating the NetID Client, it's already too late to fill in the **Callback URL** field. But I can go with the simpler sentence structure -- it's not too hard for the reader to figure out what to do.

Reply:
ok